chain-vote/src/tally.rs

```diff
@@ -1,15 +1,20 @@
+use crate::cryptography::PublicKey;
 use crate::{
     committee::*,
-    cryptography::{Ciphertext, ProofDecrypt},
+    cryptography::{Ciphertext, DleqZkp},
     encrypted_vote::SubmittedBallot,
     gang::{baby_step_giant_step, BabyStepsTable as TallyOptimizationTable, GroupElement},
 };
 use rand_core::{CryptoRng, RngCore};
-use crate::cryptography::PublicKey;
 
 /// Secret key for opening vote
 pub type OpeningVoteKey = MemberSecretKey;
 
+/// A proof of correct decryption share consists of a dleq zkp, where the committee member proves
+/// that the `DecryptionShare` is honestly derived from the `EncryptedTally` and the committee private
+/// key correspondig to its public key without disclosing it.
+pub type ProofOfCorrectShare = DleqZkp;
```

Comment:
This is where I doubt we can use this notation. As discussed, the proof of correct decryption hashes the two points of a ciphertext to compute the challenge, while a Dleq  zkp does not. So we can not directly use a Dleq proof for a proof the correct decryption described in the paper.